test/integration/stats_integration_test.cc

```diff
@@ -286,7 +286,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithFakeSymbolTable) {
   // If you encounter a failure here, please see
   // https://github.com/envoyproxy/envoy/blob/master/source/docs/stats.md#stats-memory-tests
   // for details on how to fix.
-  EXPECT_MEMORY_EQ(m_per_cluster, 44491);
+  EXPECT_MEMORY_EQ(m_per_cluster, 44715);
   EXPECT_MEMORY_LE(m_per_cluster, 46000); // Round up to allow platform variations.
```

Comment:
could you update the comment section above and explain a little bit why this bump is necessary?

Reply:
I have updated the comment. If there are a lot of callbacks in a callback manager or if adding and removing callbacks is frequent, making removal of callback handle O(1) will be a good trade off because storing a iterator is lightweight. It will make the memory consumption of a callback holder increase from 48 bytes to 56 bytes in a 64 bit linux system, but will be likely to improve performance when callback is used frequently.